frontend/public/components/factory/table-filters.ts

```diff
@@ -55,10 +55,11 @@ export const tableFilters: TableFilterMap = {
   'silence-state': (filter, silence) => filter.selected.has(silenceState(silence)),
 
   // Filter role by role kind
-  'role-kind': (filter, role) => filter.selected.has(roleType(role)),
+  'role-kind': (filter, role) => filter.selected.has(roleType(role)) || filter.selected.size === 0,
```

Comment:
This is going to be an issue for a bunch of other filters as well. We need to find a way to put the "no filters selected" logic in one place.

Reply:
@spadgett I searched the entire file for such issues. Most of them were handled previously only these two were not handled.

Reply:
However, I will search the entire console. And update here.

Reply:
We have a lot of duplicate logic in this file, though. I worry about future bugs.

I'll let it through since it's not necessarily a new problem.